### ISSUE TITLE
"CS8602: Dereference of a possibly null reference" is not raised inside tuples

### ISSUE THREAD

**Issue by ashmind** [dotnet/roslyn#41520]
**Version Used**: 
3.3.1

**Steps to Reproduce**:

```
#nullable enable
public class C {
    public object T1(string? x) => x.Length;      // CS8602
    public object T2(string? x) => (x.Length, 1); // no warning
}
```
https://sharplab.io/#gist:b3e8eeb9d0ea068a5158bf691f7e243c

**Expected Behavior**:
Two warnings `CS8602: Dereference of a possibly null reference`, one for each of `x.Length`.

**Actual Behavior**:
One warning `CS8602`, second method gives no warnings.

**Comment by jcouv** [comment]
Ouch. Will look into this asap

**Comment by jcouv** [comment]
Fixed in 16.6p1. Thanks for reporting this @ashmind
